server/queue/queue.php

```diff
@@ -39,24 +39,32 @@ function getAppointments($date, $siteId) {
 	try {
 		$canViewClientInformation = $USER->hasPermission('view_client_information');
 
-		$query = 'SELECT Appointment.appointmentId, TIME_FORMAT(AppointmentTime.scheduledTime, "%l:%i %p") AS scheduledTime, 
+		$query = 'SELECT t1.appointmentId, TIME_FORMAT(AppointmentTime.scheduledTime, "%l:%i %p") AS scheduledTime, 
 					firstName, lastName, timeIn, timeReturnedPapers, timeAppointmentStarted, timeAppointmentEnded, 
 					completed, cancelled, language, Client.clientId, 
 					(DATE_ADD(AppointmentTime.scheduledTime, INTERVAL 30 MINUTE) < NOW() AND timeIn IS NULL) AS noShow,
-					(AppointmentTime.scheduledTime < Appointment.createdAt) AS walkIn ';
+					(AppointmentTime.scheduledTime < t1.createdAt) AS walkIn , appointmentType.name as appointmentType,
+					t2.visa';
```

Comment:
I think you need a space after `visa`, so it should be `t2.visa `. We need the space b/c if we don't append the phoneNumber/emailAddress, we go straight to `FROM Appointment ...` and we need a space b/w the `visa` and `FROM`

You can also remove the space after `AS walkIn` above now

Reply:
@jcarrigan2 Actually this one probably needs to be changed before this PR can merge

Reply:
Changed it